src/main/webapp/WEB-INF/views/cars/carDrivers.jsp

```diff
@@ -0,0 +1,68 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Driver Control Panel</title>
+    <style>
+        table, th, td {
```

Comment:
for styles it is better to use a separate css file. but it's not a mistake in this task